app/src/main/java/org/mozilla/fenix/settings/creditcards/view/CreditCardEditorView.kt

```diff
@@ -49,11 +50,24 @@ class CreditCardEditorView(
         }
 
         save_button.setOnClickListener {
-            containerView.hideKeyboard()
+            saveCreditCardInfo(state)
+        }
+
+        card_number_input.text = state.cardNumber.toEditable()
+        name_on_card_input.text = state.billingName.toEditable()
+
+        bindExpiryMonthDropDown(state.expiryMonth)
+        bindExpiryYearDropDown(state.expiryYears)
+    }
+
+    @Suppress("MagicNumber")
+    internal fun saveCreditCardInfo(state: CreditCardEditorState) {
```

Comment:
This is nice. I think we should also add a helper function `getLast4CardNumber` for to return our `cardNumberLast4`. We can remove the MagicNumber if we also put the 4 inside a companion object.